docs/reference/evercrypt/randomness/index.md

```diff
@@ -0,0 +1,149 @@
+# Randomness
+
+## DRBG
```

Comment:
Let's add the header and an intro what this is implementing here as well (maybe just c&p from hacl?).